### PR TITLE
Issue 46

### PR DESCRIPTION
**Pull request by esteeroramy**

### CODE REVIEW

File: velocity.js [hunk 1]
```diff
@@ -990,13 +1088,826 @@ const handleSettingsUpdatePath = function (req, res) {
...
+            return res.status(404).render(pageNotFoundPage);
+        }
+
+        let inputTeamsList = req.body.teamsList;
+        if (!Array.isArray(inputTeamsList)) {
+            try {
+                inputTeamsList = JSON.parse(inputTeamsList);
+            }
+            catch (err) {
+                inputTeamsList = [];
```
Comment (ahmedqarmout) [inline velocity.js]: log the error

File: velocity.js [hunk 2]
```diff
@@ -990,13 +1088,826 @@ const handleSettingsUpdatePath = function (req, res) {
...
+                        if (err) {
+                            if (err.code === 6004) {
+                                team.status = common.teamStatus.ACTIVE.value;
+                                projects.addTeamToProject(projectId, team, function (err, result) {
+                                    if (err) {
+                                        logger.error(JSON.stringify(err));
+                                    }
+
+                                    updateTeamsCounter++;
+                                    if (updateTeamsCounter === resolvedTeamsList.length) {
```
Comment (ahmedqarmout) [inline velocity.js]: update the members list if the project is active

**Review by ghost — APPROVED** [review]
I need to make a lot of changes

File: velocity.js (hunk 2 above)
Comment (ahmedqarmout) [reply velocity.js]: done

File: Locales/en.json [hunk 3]
```diff
@@ -61,6 +65,61 @@
 		"selectPicture": "Select your profile picture",
 		"title": "Velocity - Profile"
 	},
+	"projects": {
+		"all": "All",
+		"form": {
+			"title": "Create a new Project",
```
Comment (esteeroramy) [inline Locales/en.json]: alphasort

File: Locales/en.json [hunk 4]
```diff
@@ -61,6 +65,61 @@
...
+			"description": "Project creation form",
+			"fields": {
+				"title": "Title",
+				"description": "Description"
+			}
+		},
+		"status": {
+			"active": "Active",
+			"draft": "Draft",
+			"closed": "Closed",
```
Comment (esteeroramy) [inline Locales/en.json]: alphasort

File: Locales/en.json [hunk 5]
```diff
@@ -61,6 +65,61 @@
...
+			"label": "Status"
+		},
+		"title": "Velocity - Projects",
+		"type": {
+			"kanban": "Kanban",
+			"scrum": "Scrum",
+			"label": "Board Type"
```
Comment (esteeroramy) [inline Locales/en.json]: alphasort

File: UI/js/common.js [hunk 6]
```diff
@@ -97,13 +127,38 @@ const translations = Object.freeze({
     mustBeCsv: 'File format must be csv!',
     mustImportOneFile: 'You can only import one file!',
     successfulFileUpload: 'File uploaded successfully',
+    emptyProjectDescription: 'Please enter your description in the editor.',
```
Comment (esteeroramy) [inline UI/js/common.js]: alphasort

File: UI/js/projects/project-page.js [hunk 7]
```diff
@@ -0,0 +1,1422 @@
...
+var groupModalHTML = null;
+var groupPrefix = '';
+var groupRow = null;
+var groupSize = null;
+var groupSelectType = null;
+var groupUserRow = null;
+var unassignedList = null;
+var projectAdminsList = null;
+var projectUsersList = null;
+var adminUserRow = null;
```
Comment (esteeroramy) [inline UI/js/projects/project-page.js]: alphasort

File: Locales/en.json (hunk 3 above)
Comment (esteeroramy) [reply Locales/en.json]: done

File: Locales/en.json (hunk 4 above)
Comment (esteeroramy) [reply Locales/en.json]: done

File: Locales/en.json (hunk 5 above)
Comment (esteeroramy) [reply Locales/en.json]: done

File: UI/js/common.js (hunk 6 above)
Comment (esteeroramy) [reply UI/js/common.js]: done

File: UI/js/projects/project-page.js (hunk 7 above)
Comment (esteeroramy) [reply UI/js/projects/project-page.js]: done

**Pull request merged**